UPGRADES.md

```diff
@@ -0,0 +1,94 @@
+
+*If using a new VPC*: 
+1. In the GitHub for [MATLAB Production Server on AWS](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws), navigate to the [MATLAB release](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) you are deploying. Under “Step 1. Launch Template,” click **Launch Stack**. (For R2023a and later, click the button under New VPC.)
+2. Fill out the template fields, then select **Create Stack**. You can monitor the resource creation from the **Stacks** section of the CloudFormation service in the AWS Portal. 
+3. Wait for the resource creation process to complete for your deployed stack. 
+
+*If using an existing VPC*: 
+1. From the GitHub for [MATLAB Production Server on AWS](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws), navigate to the [MATLAB release](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) you are deploying. Go to the FAQ section and select **Launch Stack** under “How do I use an existing VPC to deploy MATLAB Production Server?” (For R2023a and later, click the button in Step 1 under Existing VPC.)
+2. Fill out the template fields according to the GitHub documentation for this FAQ, then select **Create Stack**. You can monitor the resource creation from the **Stacks** section of the CloudFormation service in the AWS Portal. 
+3. Wait for the resource creation process to complete for your deployed stack. 
+
+### Step 2. Upload License File
+1. Log in to the Network License Manager Dashboard.
+	- *If using a new network license manager*:<ol style="list-style-type:lower-roman"><li>In the **Outputs** tab for your new stack, locate the ```MatlabProductionServerLicenseServer``` key and click the corresponding URL. This takes you to the Network License Manager for MATLAB Dashboard login page.</li><li>Log in with the username manager and the password you specified while creating the stack. </li></ol>
+	- *If using an existing network license manager*:<ol style="list-style-type:lower-roman"><li>From the EC2 instance for your network license manager, locate the public IPv4 DNS and click the corresponding URL. This takes you to the Network License Manager for MATLAB Dashboard login page.</li><li>Log in with the previously configured username and password.</li><li>Navigate to the [reference architecture documentation](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) for the MATLAB release of your new deployment. Follow the additional configuration steps in the “How to use an existing license server in an existing VPC?” subsection, under “How do I use an existing VPC to deploy MATLAB Production Server?” in the FAQ section. (For R2023a and later, follow the steps under "Use Existing License Server in Existing VPC," under "Additional Information.")</li></ol>
```

Comment:
Not sure I understand - "From the EC2 instance for your network license manager, locate the public IPv4 DNS and click the corresponding URL.". 
Do we mean locating the EC2 instance from the AWS Console?

Reply:
We can add some detail in "Log in with the previously configured username and password" e.g. Log in with the username and password that you specified when deploying the License Manager.

Reply:
Same comment as earlier re: locating "How do I use an existing VPC to deploy MATLAB Production Server?" section.